app/src/app_common.cc

```diff
@@ -164,6 +164,12 @@ const char* kCpuArchitecture = "x86";
 #else
 #error Unknown operating system.
 #endif  // Operating system
+
+#if FIREBASE_GITHUB_ACTION_BUILD
```

Comment:
This should be an `#ifdef` instead.
 
`FIREBASE_GITHUB_ACTION_BUILD` isn't defined as `1` when  the cmake variable `FIREBASE_GITHUB_ACTION_BUILD` isn't defined, and this will cause a preprocessor error in non GHA builds.  That is to say, there's no scenario where `FIREBASE_GITHUB_ACTION_BUILD` defaults to `0`, but its actually undefined instead, and so `#if FIREBASE_GITHUB_ACTION_BUILD` will fail.

Reply:
My understanding is that undefined symbols default to 0. I can add and `#else` statement to set it to 0 if you want to be explicit though. 

As a test, I just ran this, and it worked exactly as expected:
```
#include <iostream>
int main() {
#if UNDEFINED_SYMBOL
  std::cout << "Hello, world\n";
#else
  std::cout << "Goodbye, world\n";
#endif
  return 0;
}
```

```
g++ test.cpp -o test && ./test
```

```
Goodbye, world
```

Reply:
Either way I now have it set to set the symbol to 1 or 0 instead of just setting it to 1.